### PR TITLE
feat: Add 'Ask Sourcery' keybinding

### PR DESCRIPTION
**Pull request by brendanator**
Resolves SOU-1487

### CODE REVIEW

File: src/ask-sourcery.ts [hunk 1]
```diff
@@ -38,7 +38,8 @@ export function showAskSourceryQuickPick(recipes: Recipe[]) {
     }));
 
     const quickPick = vscode.window.createQuickPick();
-    quickPick.placeholder = "Ask any question or choose one of these recipes";
+    quickPick.placeholder =
+      "Ask Sourcery a question or choose one of these recipes";
```
Comment (brendanator) [inline src/ask-sourcery.ts:42]: When using the keybinding this makes it in the dialog box that it's Sourcery

File: package.json [hunk 2]
```diff
@@ -402,7 +402,17 @@
...
 				}
 			}
-		}
+		},
+    "keybindings": [
+    {
+        "command": "sourcery.chat.ask",
+        "linux": "ctrl+y",
+        "mac": "cmd+y",
+        "windows": "ctrl+shift+y",
```
Comment (brendanator) [inline package.json:411]: It would be good to test this out on windows

**Review by bm424 — APPROVED** [review]
Works well, this is quite a nice interaction. The initial message is showing at the moment but that should be fixed by sourcery-ai/core#2820 (unrelated tests failing at the moment).

File: package.json [hunk 3]
```diff
@@ -402,7 +402,17 @@
 					"description": "Show comments."
 				}
 			}
-		}
+		},
+    "keybindings": [
+    {
+        "command": "sourcery.chat.ask",
+        "linux": "ctrl+y",
```
[the remark this answers is not in the document]
Comment (bm424) [reply package.json:409]: Works On My Machine™

**Review by Hellebore — APPROVED** [review]
Works for me on mac - will test on windows when I can

**Pull request merged**